functions-marc-curtis.js

```diff
@@ -0,0 +1,82 @@
+// Write a function named marco that returns "polo".
+// const functionName = () => {
```

Comment:
The expected output is produced. The challenge question is requesting the function name to be marco.